JustSaying/Messaging/MessageProcessingStrategies/MessageConstants.cs

```diff
@@ -1,8 +1,8 @@
-﻿namespace JustSaying.Messaging.MessageProcessingStrategies
+namespace JustSaying.Messaging.MessageProcessingStrategies
 {
     public static class MessageConstants
     {
         public const int MaxAmazonMessageCap = 10;
-        public static int ParallelHandlerExecutionPerCore = 8;
+        public const int ParallelHandlerExecutionPerCore = 8;
```

Comment:
IMHO this one should be a property.

Reply:
because we might change it and not rebuild the client?

Reply:
Correct, so that if in the future we find a better value for some reason, the values aren't compiled and baked into everything that ever referenced it.

Reply:
The same logic probably also applies to `MaxAmazonMessageCap` .

Reply:
Indeed, public constants rarely need to be such.